src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -155,6 +157,33 @@ export CONDUCTR_HAPROXY_CONFIG_OVERRIDE="$CONFIG_DIR/haproxy-override.cfg"
 The `CONFIG_DIR` refers to the directory where the configuration override will be expanded. `CONDUCTR_HAPROXY_CONFIG_OVERRIDE` is the environment variable which tells ConductR where to find the custom HAProxy configuration template.
 
 
+### Specifying custom HAProxy reload script
+
+ConductR HAProxy makes use of the reload script to reload HAProxy whenever new HAProxy configuration is generated. When ConductR HAProxy starts, the reload script is copied from the path specified by `HAPROXY_RELOAD_SCRIPT_SOURCE` environment variable to the path specified by `HAPROXY_RELOAD_SCRIPT_LOCATION` environment variable.
+
+Create a file called `runtime-config.sh` within the proxy configuration directory, e.g.
+
+```lang-none
+touch /tmp/custom-haproxy-conf/runtime-config.sh
+```
+
+Populate the file with the following entry.
+
+```bash
+#!/bin/bash
+
+CONFIG_DIR=$( cd $( dirname "${BASH_SOURCE[0]}" ) && pwd )
+export HAPROXY_RELOAD_SCRIPT_SOURCE="$CONFIG_DIR/reloadHAProxy.sh"
+export HAPROXY_RELOAD_SCRIPT_LOCATION="/usr/local/etc/haproxy/reloadHAProxy.sh"
+
+# Ensure empty reload script exists with correct permission so it can be copied over
+touch "${HAPROXY_RELOAD_SCRIPT_LOCATION}"
+chmod u+x "${HAPROXY_RELOAD_SCRIPT_LOCATION}"
+```
+
+In the example above, the reload script will be copied from inside the bundle configuration to `/usr/local/etc/haproxy/reloadHAProxy.sh`. As such, prepare the `/tmp/custom-haproxy-conf/reloadHAProxy.sh`.
```

Comment:
I don't understand what you mean by "As such, prepare the `/tmp/custom-haproxy-conf/reloadHAProxy.sh`"